docs/site-replication/run-multi-site.sh

```diff
@@ -0,0 +1,236 @@
+#!/usr/bin/env bash
+
+# shellcheck disable=SC2120
+exit_1() {
+    cleanup
+    exit 1
+}
+
```

Comment:
you don't need this script we have a script called `run-multi-site-minio-idp.sh` that needs to be updated here @poornas